### PR TITLE
Support official MySQL container image.

### PR DESCRIPTION
**Pull request by kfyharukz**
Implementation for #137

### CODE REVIEW

File: e2e/manifests/mysql_cluster_external.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ spec:
             memory: "512Mi"
         livenessProbe:
           exec:
-            command: ["/ping.sh"]
+            command: ["/moco-bin/ping.sh"]
```
Comment (kfyharukz) [inline e2e/manifests/mysql_cluster_external.yaml:21]: Fix the command as `moco-agent ping`

File: e2e/Makefile [hunk 2]
```diff
@@ -61,8 +61,9 @@ build-moco-image: $(GO_FILES) ../Dockerfile
 	docker build .. -f ../Dockerfile -t controller:dev
 
 .PHONY: build-mysql-image
-build-mysql-image: $(GO_FILES) Dockerfile
-	docker build .. -f Dockerfile -t mysql:dev --build-arg MYSQL_VERSION=$(MYSQL_VERSION)
+build-mysql-image: $(GO_FILES)
```
Comment (kfyharukz) [inline e2e/Makefile:64]: Should rename target.

File: Makefile [hunk 3]
```diff
@@ -43,23 +43,16 @@ validate: setup
 .PHONY: test
 test: $(KUBEBUILDER)
 	MYSQL_VERSION=$(MYSQL_VERSION) go test -race -v -coverprofile cover.out ./...
```
Comment (masa213f) [inline Makefile:45]:
```suggestion
	MYSQL_VERSION=$(MYSQL_VERSION) go test -race -v -coverprofile cover.out ./... -ginkgo.v
```

**Review by masa213f — APPROVED** [review]
LGTM

**Pull request merged**